### ISSUE TITLE
Fix change detection

### ISSUE THREAD

**Issue by frct1** [cebor/angular-highcharts#313]
Hello,
There is possible bug: when we moving cursor on graph change detection change is calling many many times

**Comment by instantaphex** [comment]
@r-havelka No, @frct1 is correct, ALL of the highcharts events are triggering change detection, causing MASSIVE slow down.  The official angular wrapper from highcharts provides an input [runOutsideAngular] to mitigate this.  I have confirmed that change detection is running by profiling the code.

**Comment by frct1** [comment]
@instantaphex 
Nope [runOutsideAngular] in official package doesn't help

**Comment by SayakMukhopadhyay** [comment]
I am facing massive slowdowns too, but it seems to happen only on Firefox. Things are fine on Edge(Chromium) and Chrome.(Latest updates as of this post)

**Comment by jormazlop** [comment]
Hi! Is there any news or solution regarding this issue? I'm just encountering this problem, every time I move the cursor the change detection is launched continuously (And it seems that, indeed, with the official wrapper and the option [runOutsideAngular] this does not happen).
